_data/toc.yaml

```diff
@@ -27,14 +27,18 @@ guides:
     section:
     - title: "Part 1: Orientation and setup"
       path: /get-started/
-    - title: "Part 2: Containerizing an application"
+    - title: "Part 2: Build and run your image"
```

Comment:
Yes, had tried this in a previous commit. It will be quite a long entry for the table of contents. As we want the table of contents entry and the topic title to be consistent, discussed with Rob and agreed to shorten the title to `Build and run your image`.